src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -56,7 +56,10 @@ def assertEqualToDeviceOrPreloadedConditionalParam(self, attr_name, registration
       """
       attr_value = registration_request[attr_name] if attr_name in registration_request\
         else preloaded_conditionals[attr_name]
-      self.assertEqual(attr_value, record[attr_name])
+      if attr_name in ['latitude', 'longitude']:      
+        self.assertEqual(abs(attr_value-record[attr_name])<1e-6,True)
```

Comment:
It's better to use assertTrue() instead of assertEqual() in this case (the error message is easier to understand), but this technically works as well.

Reply:
Agreed. I've switched to assertTrue(). Thanks Kate.